x-pack/test/apm_api_integration/tests/services/derived_annotations.ts

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { APIReturnType } from '../../../../plugins/apm/public/services/rest/createCallApmApi';
+import { createApmApiSupertest } from '../../common/apm_api_supertest';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+import { registry } from '../../common/registry';
+
+export default function annotationApiTests({ getService }: FtrProviderContext) {
+  const supertestRead = createApmApiSupertest(getService('supertestAsApmReadUser'));
+  const es = getService('es');
+
+  const dates = [
+    new Date('2021-02-01T00:00:00.000Z'),
+    new Date('2021-02-01T01:00:00.000Z'),
+    new Date('2021-02-01T02:00:00.000Z'),
+    new Date('2021-02-01T03:00:00.000Z'),
+  ];
+
+  const indexName = 'apm-8.0.0-transaction';
+
+  registry.when(
+    'Derived deployment annotations with a basic license',
+    { config: 'basic', archives: [] },
+    () => {
+      describe('when there are multiple service versions', () => {
+        let response: APIReturnType<'GET /api/apm/services/{serviceName}/annotation/search'>;
+
+        before(async () => {
+          const { body: indexExists } = await es.indices.exists({ index: indexName });
+          if (indexExists) {
+            await es.indices.delete({
+              index: indexName,
+            });
+          }
+
+          await es.indices.create({
```

Comment:
Do you really need to create the index before calling the API? I thought that happened automatically

Reply:
If you create an index template yes, otherwise you have to create it manually right?